db/fixtures/miq_user_roles.yml

```diff
@@ -198,14 +197,37 @@
   - ems_infra_show
   - ems_infra_show_list
   - ems_infra_tag
+  - ems_infra_check_compliance
   - ems_infra_timeline
   - ems_physical_infra_console
   - ems_physical_infra_show
   - ems_physical_infra_show_list
   - ems_physical_infra_tag
   - ems_physical_infra_timeline
+  - infra_networking_view
+  - infra_networking_tag
+  - instance_view
```

Comment:
@martinpovolny since you already have instance_view, vm_view, image_view, you shouldn't need instance_compare, instance_drift, instance_timeline, instance_perf and same for image_* & vm_*